src/pages/workspace/workflows/approvals/ApprovalWorkflowEditor.tsx

```diff
@@ -0,0 +1,136 @@
+            const previousApprover = approvalWorkflow.approvers.slice(0, approverIndex).filter(Boolean).at(-1);
+            const error = approvalWorkflow?.errors?.[`approver-${approverIndex}`];
+
+            if (!error) {
+                return;
+            }
+
+            if (error === 'workflowsPage.approverCircularReference') {
+                if (!previousApprover || !approver) {
+                    return;
+                }
+                return translate('workflowsPage.approverCircularReference', {
+                    name1: approver.displayName,
+                    name2: previousApprover.displayName,
+                });
+            }
+
+            return translate(error);
+        },
+        [approvalWorkflow.approvers, approvalWorkflow.errors, translate],
+    );
+
+    const approverHintMessage = useCallback(
+        (approver: Approver | undefined, approverIndex: number) => {
+            const previousApprover = approvalWorkflow.approvers.slice(0, approverIndex).filter(Boolean).at(-1);
+            if (approver?.isInMultipleWorkflows && approver.email === previousApprover?.forwardsTo) {
+                return translate('workflowsPage.approverInMultipleWorkflows', {
+                    name1: approver.displayName,
+                    name2: previousApprover.displayName,
+                });
+            }
+        },
+        [approvalWorkflow.approvers, translate],
+    );
+
+    return (
+        <ScrollView
+            style={[styles.flex1]}
+            ref={ref}
+        >
+            <View style={[styles.mh5]}>
+                <Text style={[styles.textHeadlineH1, styles.mv3]}>{translate('workflowsCreateApprovalsPage.header')}</Text>
+
+                <MenuItemWithTopDescription
+                    title={members}
+                    titleStyle={styles.textNormalThemeText}
+                    description={translate('workflowsExpensesFromPage.title')}
+                    descriptionTextStyle={!!members && styles.textLabelSupportingNormal}
+                    onPress={() => Navigation.navigate(ROUTES.WORKSPACE_WORKFLOWS_APPROVALS_EXPENSES_FROM.getRoute(policyID, ROUTES.WORKSPACE_WORKFLOWS_APPROVALS_NEW.getRoute(policyID)))}
+                    shouldShowRightIcon
+                    wrapperStyle={[styles.sectionMenuItemTopDescription]}
+                    errorText={approvalWorkflow?.errors?.members ? translate(approvalWorkflow.errors.members) : undefined}
+                    brickRoadIndicator={approvalWorkflow?.errors?.members ? CONST.BRICK_ROAD_INDICATOR_STATUS.ERROR : undefined}
+                />
+
+                {approvalWorkflow.approvers.map((approver, approverIndex) => {
+                    const errorText = approverErrorMessage(approver, approverIndex);
+                    const hintText = !errorText && approverHintMessage(approver, approverIndex);
+                    return (
+                        <MenuItemWithTopDescription
```

Comment:
As it's allowed to edit approver, this item should be wrapped with OfflineWithFeedback. This causes a regression https://github.com/Expensify/App/issues/50477